app/src/main/java/com/apps/adrcotfas/goodtime/Main/TimerActivity.java

```diff
@@ -449,16 +449,16 @@ public void showFinishDialog(SessionType sessionType) {
         Log.i(TAG, "Showing the finish dialog.");
         AlertDialog.Builder builder = new AlertDialog.Builder(this);
         if (sessionType == SessionType.WORK) {
-            builder.setTitle("Session complete")
-                    .setPositiveButton("Start break", (dialog, which) -> start(SessionType.BREAK))
-                    .setNeutralButton("Close", (dialog, which) -> EventBus.getDefault().post(new Constants.ClearNotificationEvent()))
+            builder.setTitle(R.string.timer_activity_session_complete)
+                    .setPositiveButton(R.string.timer_activity_start_break, (dialog, which) -> start(SessionType.BREAK))
```

Comment:
There's a duplicate here and at "Start work"/ "Begin session". Please use only one of the two strings without caps in both places.